RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronSequence_cff.py

```diff
@@ -0,0 +1,88 @@
+import FWCore.ParameterSet.Config as cms
+
+#==============================================================================
+# Sequence to make low pT electrons.
+# Several steps are cloned and modified to 'open up' the sequence:
+# tracker-driven electron seeds, KF track candidates, GSF tracks.
+#==============================================================================
+
+# Low pT electron seeds
+# Below relies on default configuration for generalTracks
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronSeeds_cfi import *
+
+# Electron (KF) track candidates
+from TrackingTools.GsfTracking.CkfElectronCandidateMaker_cff import *
+TrajectoryFilterForElectronsOpen = TrajectoryFilterForElectrons.clone()
+TrajectoryFilterForElectronsOpen.minPt = 0.
+TrajectoryFilterForElectronsOpen.minimumNumberOfHits = 3
+TrajectoryBuilderForElectronsOpen = TrajectoryBuilderForElectrons.clone()
+TrajectoryBuilderForElectronsOpen.trajectoryFilter.refToPSet_ = 'TrajectoryFilterForElectronsOpen'
+electronCkfTrackCandidatesOpen = electronCkfTrackCandidates.clone()
+electronCkfTrackCandidatesOpen.TrajectoryBuilderPSet.refToPSet_ = 'TrajectoryBuilderForElectronsOpen'
+electronCkfTrackCandidatesOpen.src = 'lowPtGsfElectronSeeds'
+
+# GSF tracks
+from TrackingTools.GsfTracking.GsfElectronGsfFit_cff import *
+GsfElectronFittingSmootherOpen = GsfElectronFittingSmoother.clone()
+GsfElectronFittingSmootherOpen.ComponentName = 'GsfElectronFittingSmootherOpen'
+GsfElectronFittingSmootherOpen.MinNumberOfHits = 2
+from TrackingTools.GsfTracking.GsfElectronGsfFit_cff import * 
+electronGsfTracksOpen = electronGsfTracks.clone()
+electronGsfTracksOpen.Fitter = 'GsfElectronFittingSmootherOpen'
+electronGsfTracksOpen.src = 'electronCkfTrackCandidatesOpen'
+
+# PFTracks
+from RecoParticleFlow.PFTracking.pfTrack_cfi import *
+pfTrackOpen = pfTrack.clone()
+pfTrackOpen.GsfTrackModuleLabel = 'electronGsfTracksOpen'
+
+# PFGSFTracks
```

Comment:
@mverzett @Sam-Harper please can you check these overridden args? And check if I'm not missing any others? See: https://github.com/cms-sw/cmssw/blob/CMSSW_9_4_X/RecoParticleFlow/PFTracking/python/pfTrackElec_cfi.py